tests/test_connector_telegram.py

```diff
@@ -23,10 +23,12 @@ def setUp(self):
     def test_init(self):
         """Test that the connector is initialised properly."""
         connector = ConnectorTelegram(
-            {"name": "telegram", "token": "test"}, opsdroid=OpsDroid()
+            {"name": "telegram", "token": "test", "update-interval": 10},
```

Comment:
If I remember correctly the test will actually run through the `asyncio.sleep` and await the amount specified here. Could you please reduce this to 0.2 to keep the tests running fairly quickly?

Reply:
sure I will